pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java

```diff
@@ -271,22 +270,10 @@ public PulsarService(ServiceConfiguration config,
         // Validate correctness of configuration
         PulsarConfigurationLoader.isComplete(config);
         // validate `advertisedAddress`, `advertisedListeners`, `internalListenerName`
-        Map<String, AdvertisedListener> result =
-                MultipleListenerValidator.validateAndAnalysisAdvertisedListener(config);
-        if (result != null) {
-            this.advertisedListeners = Collections.unmodifiableMap(result);
-        } else {
-            this.advertisedListeners = Collections.unmodifiableMap(Collections.emptyMap());
```

Comment:
It looks to me that now we are no more assigning a value to this field.
Should we remove it?

Reply:
Thanks, you are right, the two of them are so alike, I made a mistake